js_modules/dagster-ui/packages/ui-core/src/asset-data/AssetHealthDataProvider.tsx

```diff
@@ -70,10 +70,50 @@ export const ASSETS_HEALTH_INFO_QUERY = gql`
     assetHealth {
       assetHealth
       materializationStatus
+      materializationStatusMetadata {
+        ...AssetHealthMaterializationDegradedPartitionedMetaFragment
+        ...AssetHealthMaterializationWarningPartitionedMetaFragment
+        ...AssetHealthMaterializationDegradedNotPartitionedMetaFragment
+      }
       assetChecksStatus
+      assetChecksStatusMetadata {
+        ...AssetHealthCheckDegradedMetaFragment
+        ...AssetHealthCheckWarningMetaFragment
+        ...AssetHealthCheckUnknownMetaFragment
+      }
```

Comment:
Is this stuff heavy? Was sort of imagining we'd fetch this on-hover but maybe it's not that big of a lift to pull the partition counts :-o 🎉